### PR TITLE
Removing HC Community Link

### PR DESCRIPTION
**Pull request by DarthHater**

### CODE REVIEW

File: search.html [hunk 1]
```diff
@@ -753,7 +753,7 @@ <h3 class="widget-title">Quick Links</h3>
                     <td><span class="MenuPointers">&gt;</span> <a href="https://groups.google.com/a/glists.sonatype.com/forum/?hl=en#%21forum/nexus-dev" target="_blank">Nexus Developer List Archive</a> (<a href="mailto:%20nexus-dev-subscribe@sonatype.org">Subscribe</a>)</td>
                   </tr>
                   <tr>
-                    <td><span class="MenuPointers">&gt;</span> <a href="https://www.hipchat.com/gW26B2y2Z" target="_blank">Live Nexus HipChat Channel</a></td>
+                    <td><span class="MenuPointers">&gt;</span> </td>
```
Comment (kellyrob99) [inline search.html]: shouldn't you be removing the entire`<tr>` since this was all that was in it?

**Comment by mosabua** [comment]
Link in the footer has to go as well.

File: search.html (hunk 1 above)
Comment (mosabua) [reply search.html]: Probably yes.

Comment (DarthHater) [reply search.html]: Good point, I'll adjust that.

**Comment by DarthHater** [comment]
Thanks @mosabua @kellyrob99 adjusted

**Comment by mosabua** [comment]
lgtm ... did you test it @DarthHater ? Or do you want to merge and see output on bamboo before you publish?

**Comment by mosabua** [comment]
Merging to verify on Bamboo

**Pull request merged**